app/src/main/res/drawable/thumb_dashlane.xml

```diff
@@ -4,6 +4,6 @@
-        android:fillColor="#007C97" />
-</vector>
\ No newline at end of file
+        android:pathData="M109.804688 42.082031C109.804688 41.320312 109.042969 40.558594 107.976562 40.253906L95.167969 35.53125C93.03125 34.617188 90.59375 35.683594 90.59375 37.210938L90.59375 90.878906C90.59375 91.640625 91.355469 92.554688 92.117188 92.859375L105.230469 97.582031C107.214844 98.34375 109.804688 97.277344 109.804688 95.597656ZM85.867188 23.328125C85.867188 22.5625 85.101562 21.800781 84.035156 21.496094L71.226562 16.773438C69.09375 15.859375 66.652344 16.925781 66.652344 18.453125L66.652344 51.078125C66.652344 51.839844 67.414062 52.753906 68.175781 53.0625L81.289062 57.785156C83.273438 58.546875 85.867188 57.480469 85.867188 55.800781ZM85.867188 82.035156C85.867188 81.273438 85.101562 80.511719 84.035156 80.207031L71.226562 75.484375C69.09375 74.570312 66.652344 75.636719 66.652344 77.164062L66.652344 109.789062C66.652344 110.550781 67.414062 111.464844 68.175781 111.769531L81.289062 116.492188C83.273438 117.253906 85.867188 116.1875 85.867188 114.507812ZM61.925781 88.59375C61.925781 87.832031 61.164062 87.066406 60.09375 86.761719L47.285156 82.039062C45.140625 81.125 42.710938 82.191406 42.710938 83.71875L42.710938 118.484375C42.710938 119.246094 43.472656 120.164062 44.234375 120.46875L57.347656 125.1875C59.332031 125.953125 61.925781 124.882812 61.925781 123.207031ZM61.925781 9.449219C61.925781 8.6875 61.164062 7.925781 60.09375 7.621094L47.285156 2.898438C45.140625 1.984375 42.710938 3.050781 42.710938 4.578125L42.710938 39.34375C42.710938 40.105469 43.472656 41.019531 44.234375 41.324219L57.347656 46.046875C59.332031 46.808594 61.925781 45.742188 61.925781 44.0625ZM37.988281 10.820312C37.988281 10.058594 37.226562 9.296875 36.15625 8.992188L23.347656 4.269531C21.214844 3.355469 18.773438 4.421875 18.773438 5.949219L18.773438 117.261719C18.773438 118.023438 19.535156 118.9375 20.300781 119.242188L33.410156 123.964844C35.394531 124.726562 37.988281 123.660156 37.988281 121.980469ZM37.988281 10.820312"
```

Comment:
This is a rather long vector path, which is bad for performance reasons. Can you split the path into separate parts? Should be fairly easy with this logo.

Reply:
To be honest I have no idea how to do this :sweat:

Reply:
I have a small VectorDrawable2Svg Python script, that generates SVGs from Android VectorDrawable format. And than in inkscape I use the "break apart" command in the path menu. This does not work for all vector logos, but in this case it does a beautiful job, because the elements are all non-overlapping.

Use the following path data for you PR:

```
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="91.03dp"
    android:height="122.87dp"
    android:viewportWidth="91.03"
    android:viewportHeight="122.87">
  <path
      android:pathData="m19.21,8.26c0,-0.77 -0.76,-1.53 -1.83,-1.83l-12.81,-4.73C2.44,0.79 0,1.86 0,3.38V114.70c0,0.76 0.76,1.67 1.53,1.98l13.11,4.72c1.98,0.76 4.57,-0.3 4.57,-1.98z"
      android:fillColor="#0e353d"/>
  <path
      android:pathData="m43.15,6.88c0,-0.76 -0.76,-1.52 -1.83,-1.82L28.51,0.33c-2.14,-0.91 -4.57,0.16 -4.57,1.68v34.77c0,0.76 0.76,1.67 1.52,1.98l13.11,4.72c1.99,0.76 4.58,-0.3 4.58,-1.98z"
      android:fillColor="#0e353d"/>
  <path
      android:pathData="m43.15,86.03c0,-0.76 -0.76,-1.53 -1.83,-1.83l-12.81,-4.73c-2.14,-0.91 -4.57,0.16 -4.57,1.68v34.77c0,0.76 0.76,1.68 1.52,1.98l13.11,4.72c1.99,0.77 4.58,-0.3 4.58,-1.98z"
      android:fillColor="#0e353d"/>
  <path
      android:pathData="m67.09,79.47c0,-0.76 -0.76,-1.52 -1.83,-1.83l-12.81,-4.72c-2.13,-0.91 -4.57,0.15 -4.57,1.68v32.62c0,0.77 0.76,1.68 1.52,1.98l13.11,4.73c1.99,0.76 4.58,-0.31 4.58,-1.99z"
      android:fillColor="#0e353d"/>
  <path
      android:pathData="m67.09,20.76c0,-0.76 -0.76,-1.52 -1.83,-1.83l-12.81,-4.72c-2.13,-0.92 -4.57,0.15 -4.57,1.68v32.62c0,0.76 0.76,1.68 1.52,1.99l13.11,4.72c1.99,0.76 4.58,-0.3 4.58,-1.98z"
      android:fillColor="#0e353d"/>
  <path
      android:pathData="m91.03,39.52c0,-0.76 -0.76,-1.53 -1.83,-1.83l-12.81,-4.72c-2.13,-0.92 -4.57,0.15 -4.57,1.68v53.66c0,0.77 0.76,1.68 1.52,1.98l13.12,4.73c1.98,0.76 4.57,-0.31 4.57,-1.99z"
      android:fillColor="#0e353d"/>
</vector>
```

Reply:
Ok thanks a lot for the tip!